### DISCUSSION TITLE
Opacity in Costume Editor

### DISCUSSION THREAD

**Discussion by thisiseiuai** [ScratchAddons/ScratchAddons#4428] [Ideas]
surprised this isn't already on there since it's in turbowarp

**Comment by CST1229** [comment]
Currently impractical to implement until a future update to Scratch that adds a color palette and changes the way colors are stored.

**Reply by thisiseiuai** [reply]
it stores it properly when moved from Turbowarp to Scratch though?

**Reply by CST1229** [reply]
Yes, but I'm referring to the way the brush color is stored, not the shape opacity.

**Reply by thisiseiuai** [reply]
the shape opacity was what I wanted to be able to edit though?

**Reply by mxmou** [reply]
> the shape opacity was what I wanted to be able to edit though?

It's a property of the fill/outline color.

**Reply by thisiseiuai** [reply]
so why not add the ability to change that property?
